src/http/verification.rs

```diff
@@ -0,0 +1,39 @@
+use html_parser::Dom;
+use std::str::FromStr;
+use url::Url;
+
+pub async fn is_verified(input_url: Url, expected_url: Url) -> bool {
+    let website = reqwest::get(input_url.to_string()).await;
+    if website.is_err() {
+        return false;
+    }
+
+    let website = website.unwrap().text().await;
```

Comment:
Curiously, is there a Rusty way to avoid the unwraps? I know you're trying to avoid levels of nesting, and you're checking for is_err on previous lines, but I always get nervous at unwraps that might panic. If I should just be OK with it given Rust programming style, all good, but I figured I'd ask since I'm trying to avoid unwrapping in general and as a result do more nesting to convince myself I handle all cases.

Reply:
One of the cases where it's okay to unwrap/expect is when you have more information than the compiler.
From the Rust book:
> It would also be appropriate to call `unwrap` or `expect` when you have some other logic that ensures the `Result` will have an `Ok` value, but the logic isn’t something the compiler understands. You’ll still have a `Result` value that you need to handle: whatever operation you’re calling still has the possibility of failing in general, even though it’s logically impossible in your particular situation. 

We've checked that the above isn't an error, then that panic can't happen :)

Reply:
I've found a clean way to avoid the unwraps :)